Formula/d/dependabot.rb

```diff
@@ -0,0 +1,31 @@
+class Dependabot < Formula
+  desc "Tool for testing and debugging Dependabot update jobs"
+  homepage "https://github.com/dependabot/cli"
+  url "https://github.com/dependabot/cli/archive/refs/tags/v1.54.0.tar.gz"
+  sha256 "ae496c65e4f947cca1251c9f786fbcc1e5e1a079af7fee08b893265e69d39295"
+  license "MIT"
+  head "https://github.com/dependabot/cli.git", branch: "main"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
+  depends_on "go" => :build
+
+  def install
+    ldflags = %W[
+      -s -w
+      -X github.com/dependabot/cli/cmd/dependabot/internal/cmd.version=#{version}
+    ]
+    system "go", "build", *std_go_args(output: bin/"dependabot", ldflags:), "./cmd/dependabot"
+
+    generate_completions_from_executable(bin/"dependabot", "completion")
+  end
+
+  test do
+    assert_match "dependabot version #{version}", shell_output("#{bin}/dependabot --version")
+    output = shell_output "#{bin}/dependabot update bundler Homebrew/homebrew 2>&1", 1
+    assert_match(/(?:Cannot|permission denied while trying to) connect to the Docker daemon/, output)
```

Comment:
maybe just do something similar like the other docker daemon prs?

Reply:
This looks pretty similar:

- https://github.com/Homebrew/homebrew-core/blob/1590d93512764bf7a821047c2f86e4995160960e/Formula/d/docker-squash.rb#L58-L62
- https://github.com/Homebrew/homebrew-core/blob/1590d93512764bf7a821047c2f86e4995160960e/Formula/d/docker-machine.rb#L37-L39
- https://github.com/Homebrew/homebrew-core/blob/1590d93512764bf7a821047c2f86e4995160960e/Formula/d/docker-buildx.rb#L46-L50

Reply:
I was looking for some common substring, and no need to use regex.

Reply:
maybe just check "connect to the Docker daemon"?

Reply:
Gotcha, let's just set `DOCKER_HOST` and match against `"Cannot connect to the Docker daemon"`.